nextflow_schema.json

```
@@ -68,6 +68,119 @@
             },
             "fa_icon": "fas fa-network-wired"
         },
+        "taxonomic_classification": {
+            "title": "Taxonomic classification",
+            "type": "object",
+            "description": "These options influence whether to activate the taxonomic classification of the input nucleotide sequences.",
+            "default": "",
+            "properties": {
+                "run_taxonomic_classification": {
+                    "type": "boolean",
+                    "description": "Activates the taxonomic classification of input mucleotide sequences.",
+                    "help_text": "This flag turns on the taxonomic classification of input nucleotide sequences. The taxonomic annotations should be turned on if the input metagenomes' bacterial sources are unknown, which can help identify the source of the AMP, BGC or ARG hit obtained for laboratory experiments. This flag should be turned off (which is by deafult) if the input nucleotide sequences represnet a single known genome or *nf-core/mag* was run beforhand. Turning on this flag relatively decreases the pipleine speed and requires >8GB RAM. Due to the size of the resulting table, the final complete summary is in a zipped format.",
```

```suggestion
                    "help_text": "This flag turns on the taxonomic classification of input nucleotide sequences. The taxonomic annotations should be turned on if the input metagenomes' bacterial sources are unknown, which can help identify the source of the AMP, BGC or ARG hit obtained for laboratory experiments. This flag should be turned off (which is by default) if the input nucleotide sequences represent a single known genome or *nf-core/mag* was run beforehand. Turning on this flag relatively decreases the pipeline speed and requires >8GB RAM. Due to the size of the resulting table, the final complete summary is in a zipped format.",
```

I've noticed this in a few places, maybe adding a VSCode spell checker  extension would be helpful?  I use: https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker